_profiles/transifex/bulk/index.md

```diff
@@ -0,0 +1,396 @@
+---
+name: Bulk
+short_description: Allow bulk operations on API endpoints
+
+extended_description: |
+  Allow multiple POST, PATCH and DELETE operation on the same endpoint with the
+  same request. Every bulk operation must be applied on the same API resource
+  collection and must be of the same type.
+
+minimum_jsonapi_version: 1.1
+
+discussion_url: http://github.com/json-api/json-api/pull/1380
+
+editors:
+  - name: Konstantinos Bairaktaris
+    email: kbairak@transifex.com
+
+categories:
+  - Bulk Operations
+---
+
+# Bulk profile
+
+## Introduction
+
+This page specifies a profile for the `application/vnd.api+json` media type,
+as described in the [JSON:API specification](http://jsonapi.org/format/).
+
+This profile allows clients to perform a number of operations with only one
+request. Instead of supporting arbitrary operations on the whole API, this
+profile limits operations of the same type (aka HTTP verb: `POST`, `PATCH`,
+`DELETE`) on the same API resource.
+
+## Creating Resources In Bulk
+
+### Requests
+
+A collection of resources can be created by sending a `POST` request to a URL
+that represents a collection of resources. The request **MUST** include an
+array of resource objects as primary data. All items of the array **MUST** have
+the same `type` member. The same restrictions that apply to a resource object
```

Comment:
What is the reason to enforce the same `type` at extension level rather than allowing implementations to decide, which resource types they support for bulk creation? I see that it will be only one resource type in most cases. But in some cases - e.g. polymorphic collections - this restriction would make it impossible to use the extension.